build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'eclipse'
 test {
   jacoco {
     excludes = ['**/package-info**','**/*Test']
-    destinationFile = file("$buildDir/reports/jacoco/test.exec")
+//    destinationFile = file("$buildDir/reports/jacoco/test.exec")
```

Comment:
Yea, I don't really like how our build system is setup right now TBH.  I am having to put a lot of effort to keep any of the jobs running.  Open to suggestions